aptos-move/move-examples/cli_args/entry_function_arguments.json

```diff
@@ -0,0 +1,30 @@
+{
+    "function_id": "<test_account>::cli_args::set_vals",
+    "type_args": [
+        "0x1::account::Account",
+        "0x1::chain_id::ChainId"
+    ],
+    "args": [
+        {
+            "arg_type": "u8",
+            "arg_value": 123
```

Comment:
@banool originally I was hoping to do that, but unfortunately `type` isn't supported as a struct field name for `ArgWithTypeJSON`: https://github.com/aptos-labs/aptos-core/blob/a6d6be73ed/crates/aptos/src/common/types.rs#L1931

Hence rather than `arg_type` and `value`, I went with `arg_type` and `arg_value` for consistency.

Your thoughts?

Reply:
It should be possible with something like this: https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos-rosetta/src/types/objects.rs#LL168C22-L168C22.

Reply:
https://github.com/aptos-labs/aptos-core/blob/6aa74479f6b3f2b44f06cc0aea7efbd9acff1a51/crates/aptos/src/common/types.rs#L1919